src/providers/candidate-details-check/__mocks__/candidate-details-check.mock.ts

```diff
@@ -0,0 +1,6 @@
+export class CandidateDetailsCheckProviderMock {
+
+  isCandidateDetailsSeen(slotId: number): boolean {
```

Comment:
This isn't a testable mock

` isCandidateDetailsSeen = jasmine.createSpy('isCandidateDetailsSeen').and.returnValue(true)